src/plat/rk3288/config.cmake

```diff
@@ -24,13 +24,13 @@ foreach(config IN LISTS cmake_configs)
 endforeach()
 
 if(KernelPlatformRK3288)
-#    if("${KernelSel4Arch}" STREQUAL aarch32)
-     declare_seL4_arch(aarch32)
-#    elseif("${KernelSel4Arch}" STREQUAL arm_hyp)
-#        declare_seL4_arch(arm_hyp)
-#    else()
-#        fallback_declare_seL4_arch_default(aarch32)
-#    endif()
+    #("${KernelSel4Arch}" STREQUAL aarch32)
```

Comment:
Are the lines commented out still useful?

Reply:
They will be.  We'll be adding arm-hyp support in the very near future.